src/Assets/ManageDates/Templates/DateLocalization-01.json

```diff
@@ -797,6 +797,12 @@
           "Description": "",
           "DisplayFolders": ""
         },
+        {
+          "OriginalName": "Quarter",
+          "Name": "Quartal",
+          "Description": "",
+          "DisplayFolders": ""
+        },
```

Comment:
@albertospelta thanks for replying. 

I am currently modifying "DateLocalization-01.json" file to add Quarter translation for IT/FR/ES/PT. It seems that Quarter is translated to "Trimestre" in all the 4 languages, interesting fact to learn. :-) I used google translation + deepl.com + the original translation from other contributors for triple check.. I hope it is correct. ( I was changing only German because I work now in Germany and I can speak German. :-) )

For DateLocalization-02.json file, I am almost done with the additional columns and hierarchies translation.
However, would like shortly clarify one small quesion before I upload.

It is about the column name under the hierarchy structure. It is confusing for me because the column names in the hierarchy don’t seem to map with the real column. See below screenshot. What will be your comments on this question ?
Any specific logic why it was designed/named so in the hierarchy ?

Sorry for bothering again. thanks a lot in advance.

![图片](https://user-images.githubusercontent.com/51952216/174455852-799d36ce-e2e8-4810-97c8-fab370c86db9.png)

![图片](https://user-images.githubusercontent.com/51952216/174455893-07b3515c-ca3d-443d-b486-3536f5a5662e.png)

Reply:
I understand this can confuse the developer but it is only a data modeling convention which in my opinion makes the hierarchy easier to understand when developing a report. I think the approach in the last image below is more readable for the final user.

![image](https://user-images.githubusercontent.com/1957788/174743115-505e3561-a958-4422-9621-1c61e6731ea4.png)

![image](https://user-images.githubusercontent.com/1957788/174743320-bb5041e7-44c5-4499-af1c-5e5632dd5e97.png)